pkg/tsdb/azuremonitor/tokenprovider/token_provider_azure.go

```diff
@@ -1,4 +1,4 @@
-package pluginproxy
```

Comment:
Can we call this just token_provider.go? The azure prefix is given by the context I think.